src/assets/styles/global.scss

```diff
@@ -51,22 +68,30 @@
 
 .calcite-theme-dark {
   @include calcite-theme-dark-extended();
+  animation-name: calciteThemeChangeDark;
 }
 
 .calcite-theme-light {
   @include calcite-theme-light-extended();
+  animation-name: calciteThemeChangeLight;
 }
 
 .calcite-theme-auto {
   @media (prefers-color-scheme: dark) {
     @include calcite-theme-dark-extended();
+    animation-name: calciteThemeChangeDark;
   }
 
   @media (prefers-color-scheme: light) {
     @include calcite-theme-light-extended();
+    animation-name: calciteThemeChangeLight;
   }
 }
 
+body:not(.calcite-theme-dark):not(.calcite-theme-auto):not(.calcite-theme-light) {
```

Comment:
Is this meant exclusively for `body` or anywhere a theme can be applied? I believe `animationstart` bubbles, so this could lead to some boilerplate on end-users.

Reply:
Yeah this part is only meant to work on `body`. The main scenario for dynamically changing themes is for the full app. This handles the default scenario when no theme class is set and it defaults to light. 

If people are dynamically changing themes for different parts of their UI - that seems concerning right? But even if there is a valid use case, they can explicitly add remove the dark/light themes and it will work fine. I'm going to document all of that here (until we find a better place):
https://developers.arcgis.com/calcite-design-system/faq/#how-do-i-change-themes

Reply:
For the JSAPI use case, we can just listen for the animation change on body right? I don't think we need to be concerned about themes changing on any other elements?

Reply:
> For the JSAPI use case, we can just listen for the animation change on body right? I don't think we need to be concerned about themes changing on any other elements?

Correct